app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/base/BaseRealm.kt

```diff
@@ -0,0 +1,12 @@
+package com.puzzlebench.clean_marvel_kotlin.presentation.base
+
+import android.app.Application
+import io.realm.Realm
+
+class BaseRealm() : Application() {
```

Comment:
rename as BaseRealmApplication pls and put this in the package root, in presentation not inside the base package

Reply:
Done